src/app.module.ts

```diff
@@ -1,17 +1,12 @@
 import { Module } from '@nestjs/common';
 import { ConfigModule } from '@nestjs/config';
-import { TypeOrmModule } from '@nestjs/typeorm';
 import { getEnvPath } from './common/helpers/env.helper';
-import { TypeOrmConfigService } from './typeorm/typeorm.service';
 import { ApiModule } from './api/api.module';
+import { DatabaseModule } from './typeorm/typeorm.module';
 
 const envFilePath: string = getEnvPath(`${__dirname}/common/envs`);
 
 @Module({
-  imports: [
-    ConfigModule.forRoot({ envFilePath, isGlobal: true }),
-    TypeOrmModule.forRootAsync({ useClass: TypeOrmConfigService }),
-    ApiModule,
-  ],
+  imports: [ConfigModule.forRoot({ envFilePath, isGlobal: true }), DatabaseModule, ApiModule],
```

Comment:
Won't the application need to know about the TypeOrmModule? Why remove it here?

Reply:
@JeromeGoodrich I added it to the DatabaseModule to clean it up a bit, and keep the modules organized.